libs/feature/search/src/lib/utils/mapper/elasticsearch.mapper.spec.ts

```diff
@@ -150,13 +150,15 @@ describe('ElasticsearchMapper', () => {
             hit._source.link = [
               {
                 protocol: 'MY-PROTOCOL',
-                name: 'my data layer',
-                url: 'https://my.website/services/data/',
+                nameObject: { default: 'my data layer' },
+                urlObject: { default: 'https://my.website/services/data/' },
+                descriptionObject: { default: '' },
```

Comment:
Although it's nice that you modified the tests to cover the new logic, here it's also important to make sure that the old logic still works, especially when some fields are missing (name and/or description).

I would advise that you create two new test cases covering the case where the url, name and description are multilingual: one with all the properties (`urlObject`, `nameObject` and `descriptionObject`), and another one with just `urlObject`.

Reply:
nice! as a general rule of thumb, we should always be very careful when modifying existing tests; after all, they're here to guarantee that a code still behaves properly after a change! :slightly_smiling_face: